nf_core/modules/nfcore_module.py

```diff
@@ -38,12 +38,13 @@ def __init__(self, module_name, repo_url, module_dir, repo_type, base_dir, nf_co
         self.is_patched = False
         self.is_patched = None
```

Comment:
I just noticed this duplication of the `is_patched` attribute. Not sure which one should stay?

Reply:
Seems like `False` should be the one that makes more sense, as it's supposed to be a bool.